aws/README.md

```diff
@@ -6,16 +6,31 @@
 3. [AWS IAM Authenticator](https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html)
 
 ### Set Your Variables
-1. Clone this repository to your local machine.
-2. Uncomment and replace variables in `terraform.tfvars` with the desired values. Check `variables.tf` for default values.
+1. Fork and clone this repository to your local machine.
```

Comment:
Haven't we already cloned at this point (should be handled by root README?)

Reply:
I chose to clarify this step rather than removing it altogether. "If you haven't already done so,...". That way we account for the user who _has_ already cloned, and the one who's still perusing the repo.